ext/drmgr/drmgr.c

```diff
@@ -215,6 +221,12 @@ static cb_list_t cblist_app2app;
 static cb_list_t cblist_instrumentation;
 static cb_list_t cblist_instru2instru;
 
+/* For opcode specific instrumentation. */
+static hashtable_t global_opcode_instrum_table; /* maps opcodes to cb lists */
+static void *opcode_table_lock;
+/* a flag indicating whether opcode instrum was ever registered */
+static bool was_opcode_instrum_registered;
```

Comment:
Comment that this is protected by bb_cb_lock

Reply:
Done.